### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by nilesh02** [ugnelis/tensorflow-rnn-ctc#18]
Is this code anywhere using batch normalization?

**Comment by ugnelis** [comment]
Hey @nilesh02, batch normalization is not implemented. Open for Pull Requests :)
